README.md

```diff
@@ -58,28 +62,32 @@ It will create a `db` folder containing the local vectorstore. Will take 20-30 s
 You can ingest as many documents as you want, and all will be accumulated in the local embeddings database. 
```

Comment:
```suggestion
You can ingest as many documents as you want, and all will be accumulated in the local embeddings database.
```
Nit:
Since you are fixing these.